src/commands.ts

```diff
@@ -77,6 +83,16 @@ export function registerCommands() {
         updateFileAccess(FileAccess.ReadOnly);
     }));
 
+    Container.context.subscriptions.push(commands.registerCommand("readOnly.makeWriteableRecursive", (uri?: Uri) => {
+        if(!uri) return;
```

Comment:
As suggested, make the command work for files and folder. When choosing a folder, ask/warn the user about being recursive.